object/virtual_device_list_test.go

```diff
@@ -762,6 +762,20 @@ func TestCreateSCSIController(t *testing.T) {
 	}
 }
 
+func TestCreateSATAController(t *testing.T) {
+	for _, l := range []VirtualDeviceList{SATAControllerTypes(), devices} {
+		_, err := l.CreateSATAController()
+		if err != nil {
+			t.Error("should fail")
```

Comment:
Looks like some copy-n-pasta from other tests, but can be removed here.

Reply:
Resolved in 2e77836.